src/drive/web/modules/views/OnlyOffice/Title.jsx

```diff
@@ -1,34 +1,32 @@
-import React from 'react'
-import { makeStyles } from '@material-ui/core/styles'
+import React, { useContext } from 'react'
 
 import { DialogTitle } from 'cozy-ui/transpiled/react/Dialog'
 import Divider from 'cozy-ui/transpiled/react/MuiCozyTheme/Divider'
 
+import SharingBanner from 'drive/web/modules/public/SharingBanner'
+import { useSharingInfos } from 'drive/web/modules/public/useSharingInfos'
+import { OnlyOfficeContext } from 'drive/web/modules/views/OnlyOffice'
 import Toolbar from 'drive/web/modules/views/OnlyOffice/Toolbar'
 
-const useStyles = makeStyles(() => ({
-  root: {
-    display: 'flex',
-    alignItems: 'center',
-    height: '48px',
-    padding: '0 1rem'
-  }
-}))
+import styles from './styles.styl'
 
 const Title = () => {
-  const styles = useStyles()
+  const { isPublic } = useContext(OnlyOfficeContext)
+  const sharingInfos = useSharingInfos()
 
   return (
     <>
       <DialogTitle
         data-testid="onlyoffice-title"
         disableTypography
-        className="u-ellipsis"
-        classes={styles}
+        className={`${
+          styles['title']
+        } u-ellipsis u-flex u-flex-items-center u-p-0 u-pr-1`}
```

Comment:
nit: you can use cx(styles['title'], 'u-ellipsis ...) instead template.

Reply:
Oui mais comme il n'y a pas de condition à gérer ici, j'ai préféré ne pas utiliser une lib pour rien... à moins que ? Qu'est-ce qu'apporterait `cx` de plus ici ?

Reply:
Non, juste une habitude